config/base/params.env

```diff
@@ -1 +1,4 @@
 odh-model-controller=quay.io/opendatahub/odh-model-controller:fast
+caikit-tgis-image=quay.io/opendatahub/caikit-tgis-serving:fast
+tgis-image=quay.io/opendatahub/text-generation-inference:fast
+ovms-image=quay.io/opendatahub/openvino_model_server:fast
```

Comment:
take https://github.com/opendatahub-io/odh-dashboard/blob/main/manifests/overlays/modelserving/kustomization.yaml for example, these 3 new images wont be necessary in the parmas.env

Reply:
AFAIK, this is needed for konflux and it was requested by devOps team.

Reply:
Hi @zdtsw , sorry I should have added more description to PR. We are migrating templates to odh-model-controller and we were asked by DevOps team to move the images to params.env --> for disconnected, they use a script that picks the params.env from all other manifests (except us right now, hence the change).

Reply:
Plus as JooHo mentioned, it probably will be needed for konflux as well. We are in the process of migrating TGIS, Caikit-TGIS and OVMS builds to konflux as well (it might span a few sprints though).